lucille-core/src/main/java/com/kmwllc/lucille/stage/util/StreamGobbler.java

```diff
@@ -0,0 +1,60 @@
+package com.kmwllc.lucille.stage.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A general stream gobbler, useful when starting processes and handling their streams
+ */
+public class StreamGobbler extends Thread {
+  protected transient InputStream processOut;
+  protected transient OutputStream processIn;
+  protected String name;
```

Comment:
where is this name used? threads have names, and we're extending thread, so can call getName() if we need the name that was passed into the superconstructor